### PR TITLE
Check for presence of home address and cellphone

### LINKED ISSUE #343: Create validation tests for student model's cell_phone and home_address attributes

### LINKED ISSUE #344: Create validations for student model's  cell_phone and home_address attributes

### PR DESCRIPTION
**Pull request by sonia-y**
Fix #344 
Fix #343

### CODE REVIEW

File: app/models/student.rb [hunk 1]
```diff
@@ -24,6 +24,8 @@ class Student < ActiveRecord::Base
   validates :email, presence: true
   validates :first_name, presence: true
   validates :last_name, presence: true
+  validates :home_address, presence: true
+  validates :cell_phone, presence: true
```
Comment (warrenshen) [inline app/models/student.rb]: Can you alphabetize these?

File: spec/models/student_spec.rb [hunk 2]
```diff
@@ -39,4 +39,13 @@
...
 
+  it "is invalid without a cell_phone" do
+    factory = FactoryGirl.build(:student, cell_phone: nil)
+    expect(factory).to be_invalid
+  end
+
+  it "is invalid without a home_address" do
+    factory = FactoryGirl.build(:student, home_address: nil)
+    expect(factory).to be_invalid
+  end
```
Comment (warrenshen) [inline spec/models/student_spec.rb]: Can you alphabetize these as well!

File: app/models/student.rb [hunk 3]
```diff
@@ -21,9 +21,13 @@ class Student < ActiveRecord::Base
 
   has_many :comments, dependent: :destroy
 
+  validates :cell_phone, presence: true
   validates :email, presence: true
   validates :first_name, presence: true
+  validates :home_address, presence: true
   validates :last_name, presence: true
+
```
Comment (maxwolffe) [inline app/models/student.rb]: Would you remove this extra whitespace?

File: spec/models/student_spec.rb [hunk 4]
```diff
@@ -34,9 +39,14 @@
...
+
   it "is invalid without a last_name" do
     factory = FactoryGirl.build(:student, last_name: nil)
     expect(factory).to be_invalid
   end
-
+
```
Comment (maxwolffe) [inline spec/models/student_spec.rb]: Would you remove this extra whitespace?

**Pull request merged**